### PR TITLE
update smoking cessation tools to include modal pop overs

### PR DESCRIPTION
**Pull request by cbitstech**
[#119226733]

### CODE REVIEW

File: src/partials/session_sub_nav.html [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+        </a>
+      </div>
     </div>
-</div>
\ No newline at end of file
+  </div>
+</div>
+
+<script type="text/javascript">
+  $("#cessation_date_input").mobiscroll().date();
```
Comment (eschlange) [inline src/partials/session_sub_nav.html]: Does this always set the #cessation_date_input value to the current date? Should this dynamically pull in an existing date if it exists?

**Comment by eschlange** [comment]
@cbitstech aside from my one question, everything looks good. Please merge if the date loading logic is as intended.

File: src/partials/session_sub_nav.html (hunk 1 above)
Comment (cbitstech) [inline src/partials/session_sub_nav.html]: It should, you are absolutely right, cancelling merge.

File: src/js/controllers/SessionSubNavController.js [hunk 2]
```diff
@@ -1,50 +1,56 @@
...
+    this.getCessationDate = function () {
+      return configurationService.getCessationDate();
+    };
+
+    this.setCessationDate = function () {
+      configurationService.saveCessationDate(this.cessationDate);
+    };
+
+    this.openRiskyTimesModal = function () {
+      configurationModalService.openRiskyTimesModal($modal);
```
Comment (ericcf) [inline src/js/controllers/SessionSubNavController.js]: Is there a reason this can't be accomplished with a directive?

Comment (cbitstech) [inline src/js/controllers/SessionSubNavController.js]: What specifically?

Comment (ericcf) [inline src/js/controllers/SessionSubNavController.js]: Manipulating the DOM. Ideally, Angular directives should be used for that: https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#manipulate-dom-in-a-directive

Comment (cbitstech) [inline src/js/controllers/SessionSubNavController.js]: This is code that has been refactored/repurposed from the setup controller--I don't have the time resources to refactor it into a directive, but could definitely see the value moving forwards.

**Comment by ericcf** [comment]
What's going on with this PR? Does it still need review, can it be closed, or will it be updated further? Thanks.

**Comment by cbitstech** [comment]
I don't think it makes sense to turn this into a directive given the resources on this project.

**Comment by cbitstech** [comment]
I'm taking this as my Saturday challenge to do this in less than 1 hour. Wish me luck!

**Comment by cbitstech** [comment]
I'm issuing a new pull request and deleting this one.

**Pull request closed without merging**